### PR TITLE
added hrf logo in nav and in footer, and added images to assets

### PR DESCRIPTION
**Pull request by nickcabuno**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My code has been reviewed by at least one peer
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
- [x ] There are no merge conflicts

### CODE REVIEW

**Review by Iamlegend-Imani — APPROVED** [review]

**Review by jossters — APPROVED** [review]
awesome work on getting approval

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ import React, { useState } from 'react';
 import 'antd/dist/antd.css';
 import { Link, NavLink } from 'react-router-dom';
 
-import logo from '../../assets/blue-witness1.png';
```
Comment (RyanFikejs) [inline src/components/NavBar/NavBar.js:5]: This is good to swap this out for clarity in labeling which logo we are using. For my sake, please go through and doublecheck that all occurences of 'logo' are swapped out for 'bwlogo'.

**Review by RyanFikejs — APPROVED** [review]
Good job adding this in, have fun and come up with a couple of mockups for placement of the item (above, below, etc.) to present to the stakeholder for approval (and consult with your UI/UX buddies for consensus).

File: src/components/NavBar/NavBar.js (hunk 1 above)
Comment (nickcabuno) [reply src/components/NavBar/NavBar.js:5]: getting on it right now

**Pull request merged**